mod/profile.php

```diff
@@ -181,6 +181,10 @@ function profile_content(&$a, $update = 0) {
 		$commpage = (($a->profile['page-flags'] == PAGE_COMMUNITY) ? true : false);
 		$commvisitor = (($commpage && $remote_contact == true) ? true : false);
 
+		if(feature_enabled($a->profile['profile_uid'],'photos_widget')) {
+			require_once('include/photos.php');
```

Comment:
I would prefer to have `require_once()`  always on top of the page, not in the code